### PR TITLE
Feature/mat 7098 valueset action center

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE PR

Jira Ticket: [MAT-7098](https://jira.cms.gov/browse/MAT-7098)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/CqlBuilderPanel/ValueSets/Results/Results.tsx [hunk 1]
```diff
@@ -179,23 +204,44 @@ export default function Results(props: ResultsProps) {
...
-                <div className="chevron-container">
-                  <ExpandMoreIcon sx={{ color: "#0073c8" }} />
-                </div>
-              </button>
-            </div>
+                <ControlPointIcon color="primary" />
+              </ToolTippedIcon>
+              <Box sx={{ flexGrow: 1 }} />
+              <ActionCenter
+                canEdit={true}
```
Comment (adongare) [inline src/CqlBuilderPanel/ValueSets/Results/Results.tsx:235]: is this hardcoded because search is visible only for owned draft measures?

Comment (nmorasb) [reply src/CqlBuilderPanel/ValueSets/Results/Results.tsx:235]: yes, but also, I just realized I don't think I wired up canEdit within the action center component. So doing that now and adding a couple tests

Comment (nmorasb) [reply src/CqlBuilderPanel/ValueSets/Results/Results.tsx:235]: I'll be removing the canEdit in a future PR. The results grid will only show for users with Edit action, so the action center will only be visible for edit users in this instance.
I will ask Mike, but I think we generally try to lean away from showing disabled buttons/icons if there is nothing the user can do to enable them. This means, actions that a user can never perform (for lack of edit permissions) just shouldn't even be included in the action center, which means any component using the action center would be responsible for determining what actions should be present based on permissions.

Comment (adongare) [reply src/CqlBuilderPanel/ValueSets/Results/Results.tsx:235]: sounds good. Thanks Nate!

**Review by adongare — APPROVED** [review]
